generators/client/templates/vue/src/main/webapp/app/account/principal.ts.ejs

```diff
@@ -34,6 +34,16 @@ export default class Principal {
     if (!this.store.getters.account && !this.store.getters.logon && token) {
       this.retrieveAccount();
     }
+    this.retrieveProfiles();
+  }
+
+  public retrieveProfiles(): any {
+    axios.get('management/info').then((res) => {
+    if (res.data && res.data.activeProfiles && res.data.activeProfiles.indexOf(res.data['display-ribbon-on-profiles']) > -1) {
```

Comment:
indent: need 2 additional spaces

Reply:
Yes please, fix indentation and then we could merge this !

Reply:
done